core/src/main/java/org/springframework/security/authentication/DelegatingReactiveAuthenticationManager.java

```diff
@@ -18,7 +18,10 @@
 
 import java.util.Arrays;
 import java.util.List;
+import java.util.function.Function;
```

Comment:
Please update the copyright header to end with `2024`

Reply:
Done.